addons/account_edi/models/account_edi_document.py

```diff
@@ -19,7 +19,7 @@ class AccountEdiDocument(models.Model):
     _description = 'Electronic Document for an account.move'
 
     # == Stored fields ==
-    move_id = fields.Many2one('account.move', required=True, ondelete='cascade')
+    move_id = fields.Many2one('account.move', required=True, ondelete='cascade', index='btree_not_null')
```

Comment:
A `required=True` field cannot contains NULLs, it should be a simple `index=True`

Reply:
Fixed